contentcuration/contentcuration/tests/test_contentnodes.py

```diff
@@ -232,24 +231,16 @@ def test_move_nodes(self):
         new_channel.main_tree.get_children().delete()
         new_channel_node_count = new_channel.main_tree.get_descendants().count()
 
-        nodes = []
-
-        for node in self.channel.main_tree.get_children():
-            nodes.append({'id': node.pk})
-
-        assert self.channel.main_tree.pk not in [node['id'] for node in nodes]
-
         # simulate a clean, right-after-publish state for both trees to ensure they are marked
         # changed after this
         self.channel.main_tree.changed = False
         self.channel.main_tree.save()
         new_channel.main_tree.changed = False
         new_channel.main_tree.save()
 
-        move_nodes(new_channel.id, new_channel.main_tree.id, nodes,
-                   min_order=0, max_order=len(nodes))
+        for node in self.channel.main_tree.get_children():
+            node.move_to(new_channel.main_tree)
```

Comment:
Is the `move_nodes` function no longer being used? If so, we may want to remove it (and any task code that calls it). If we're leaving it in the code, though, I think it's good to have a test that calls it, so I'd suggest making this a separate test instead. (Or having the test try both methods.)

Reply:
It gets removed in the follow up PR to this one https://github.com/learningequality/studio/pull/2335/files#diff-f522fb0a487580b66291c9ae6b2b642dL330 - I didn't remove it in this PR in order to limit the scope of the diff.